BackEnd/utils/send_email.py

```diff
@@ -0,0 +1,80 @@
+from datetime import timedelta
+import os
+from email.mime.image import MIMEImage
+from django.utils.timezone import now
+from django.core.mail import EmailMultiAlternatives
+from django.conf import settings
+from django.template.loader import render_to_string
+from administration.models import AutoModeration
+from .image_moderation import ModerationManager
+
+
+def define_ending(hours):
+    result_of_hours = None
+    if hours % 10 == 1 and hours != 11:
+        result_of_hours = f"{hours} годину"
+    elif hours % 10 in range(2, 5) and hours != 12:
+        result_of_hours = f"{hours} години"
+    else:
+        result_of_hours = f"{hours} годин"
+    return result_of_hours
+
+
+def attach_image(email, image, content_id):
+    image_name = os.path.basename(image.image_path.name)
+    with open(image.image_path.path, "rb") as img_file:
+        img = MIMEImage(img_file.read(), _subtype=image.content_type)
+        img.add_header("Content-ID", f"<{content_id}>")
+        img.add_header(
+            "Content-Disposition", f'inline; filename="{image_name}"'
+        )
+        email.attach(img)
+
+
+def send_moderation_email(profile):
+    manager = ModerationManager(profile)
+    if manager.check_for_moderation():
+        update_datetime = now() + timedelta(hours=3)
```

Comment:
If I understand correctly this row responsible for timezone shift, because our timezone is +3 now, am i right?
If yes so when you finish development part and move project to productions so you should spent some time twice a year to change this code line because we have +2 an +3 depends of part of the year. And this is bad idea, our backend should work with own time.

Reply:
We decided to keep the time in UTC.